### PR TITLE
Add fault tolerance table ObolNetwork/obol-docs#114

### PR DESCRIPTION
**Pull request by OisinKyne**
## Details

ticket:

### CODE REVIEW

File: docs/int/key-concepts.md [hunk 1]
```diff
@@ -85,6 +85,16 @@ A distributed validator cluster is a collection of distributed validator nodes c
...
 
+| Cluster Size | Threshold | Note              |
+|:------------:|:---------:|:------------------|
+| 4            |  3/4      | Minimum threshold |
+| 5            |  4/5      |                   |
+| 6            |  4/6      |                   |
+| 7            |  5/7      |                   |
+| 8            |  6/8      |                   |
+| 9            |  6/9      |                   |
+| 10           |  7/10     |                   |
```
Comment (OisinKyne) [inline docs/int/key-concepts.md:96]:
```suggestion
| 4            |  3/4      | Minimum threshold |
| 5            |  4/5      |                   |
| 6            |  4/6      | Minimum to tolerate two offline nodes|
| 7            |  5/7      | Minimum to tolerate two **malicious** nodes |
| 8            |  6/8      |                   |
| 9            |  6/9      | Minimum to tolerate three offline nodes |
| 10           |  7/10     | Minimum to tolerate three **malicious** nodes |
```

File: docs/int/key-concepts.md [hunk 2]
```diff
@@ -85,6 +85,16 @@ A distributed validator cluster is a collection of distributed validator nodes c
 
 A distributed validator key is a group of BLS private keys, that together operate as a threshold key for participating in proof of stake consensus with.
 
+| Cluster Size | Threshold | Note              |
```
Comment (OisinKyne) [inline docs/int/key-concepts.md:88]:
```suggestion
The number of nodes in a cluster that need to be online and honest for their distributed validators to be online is outlined in the following table.

| Cluster Size | Threshold | Note              |
```

Can't tell whether this should be an *is* outlined in the following or *are* outlined in the following...

**Comment by OisinKyne** [comment]
@EridianAlpha left my suggestions in the form of a PR

**Comment by OisinKyne** [comment]
And actually, it would be good if we give this table a `###` title such that we can have an anchor tag for it to link places. So maybe move it just after `### Distributed Validator Key Share` and call the section `Distributed Validator Threshold`.

File: docs/int/key-concepts.md (hunk 2 above)
Comment (EridianAlpha) [reply docs/int/key-concepts.md:88]: I think "is" makes sense in this case.

**Pull request merged**